model/map/src/main/java/org/keycloak/models/map/storage/ModelCriteriaBuilder.java

```diff
@@ -61,10 +61,10 @@
      * is now.
      */
     public enum Operator {
-        /** Equals to */
-        EQ,
-        /** Not equals to */
-        NE,
+        /** Equals to for single value objects and contains the value for list objects */
```

Comment:
```suggestion
        /**
         * This single-parameter operator checks that the property has the given value: <ul>
         * <li>In case of single values, this corresponds to equality check to the operand</li>
         * <li>In case of lists, this corresponds to checking that this list contains an object equivalent to the operand</li>
         * </ul>
         */
         
```